src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -464,6 +464,35 @@ in order to see the theme color and favicon changes, open the
   });
 ```
 
+## Debugging and testing dark mode
+
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so in Chrome DevTools, we now allow you to emulate the user's preferred color scheme
```

Comment:
This line should be removed now (sorry, GitHub wouldn't let me suggest a multi-line patch before)

Reply:
Ooops, fixed. Thanks!